README.md

```diff
@@ -1,7 +1,8 @@
 # Medic Mobile
 
-These instructions should help you get setup to run or develop on Medic Mobile.
-For latest changes and release announcements see our [change log](Changes.md).
+These instructions should help you get setup to run or develop on Medic Mobile's Community Health Application Framework. For latest changes and release announcements see our [change log](Changes.md).
```

Comment:
Changes.md is the old location - now it just has a link to the new location. I think change this to...

`...[release notes](https://github.com/medic/medic/tree/master/release-notes).`